swarm/network/stream/stream.go

```diff
@@ -421,7 +421,7 @@ func (r *Registry) updateSyncing() {
 			delete(streams, stream)
 			delete(streams, getHistoryStream(stream))
 		}
-		err := r.RequestSubscription(p.ID(), stream, NewRange(0, 0), Top)
+		err := r.RequestSubscription(p.ID(), stream, NewRange(0, 0), High)
```

Comment:
Can you explain a bit why we are reducing the priority?

Reply:
I can see that we don't use `Mid` and `Low`, so we are left only with two different priorities.

Reply:
yes @nonsense if we are syncing while we download, if the requests and deliveries are sent alongside syncing messages, then they cannot jump queue as it were. 
in other words we do not want the syncstream to stall a request or request response chunk delivery, therefore retrievals via DPA has higher priority which means lower priority is sent only no higher once ! intermingling sync deliveries and request deliveries

Reply:
thanks for the explanation.